src/mpid/ch4/shm/xpmem/xpmem_win.c

```diff
@@ -49,7 +48,7 @@ static int get_node_ranks(MPIR_Comm * shm_comm_ptr, int *shm_ranks, int *node_ra
     goto fn_exit;
 }
 
-int MPIDI_XPMEM_mpi_win_create_hook(MPIR_Win * win)
+int MPIDI_XPMEM_on_win_create(MPIR_Win * win)
 {
```

Comment:
I'd keep the `_hook` suffix because this is not a full win creation. Same feeling for other routines.

Reply:
I thought the `_hook` naming is to conform to the netmod/shmmod api interface, and in this case, since xpmem is not really a full shmmod, I felt it helps to differentiate them. Part of the effort in this PR is make xpmem less like a shmmod, but more like an independent utility or protocol module so any shmmod can use.